### ISSUE TITLE
socks5/https proxy support please

### ISSUE THREAD

**Issue by fisherwei** [hellcatz/hminer#1]
most unified threat management/next-gen firewall/layer7 firewall can block mining

**Comment by hellcatz** [comment]
To have encrypted connection between the stratum miner -> proxy -> pool:
Use a stratum proxy: <https://github.com/hellcatz/verusProxy>

**Comment by alexgabbard01** [comment]
> To have encrypted connection between the stratum miner -> proxy -> pool: Use a stratum proxy: https://github.com/hellcatz/verusProxy

I don't think you understood that he talks about Upstream Socks5/HTTP/S proxy, not stratum proxy.

**Comment by Kabingeran** [comment]
@hellcatz Need socks5 proxy support
